spec/liquid_tags/gist_tag_spec.rb

```diff
@@ -12,6 +12,7 @@
 
     let(:bad_links) do
       [
+        nil,
```

Comment:
`bad_links` is only used in the following spec:

```ruby
    it "rejects XSS attempts" do
      bad_links.each do |link|
        expect { generate_new_liquid(link) }.to raise_error(StandardError)
      end
    end
```

This is not specifically related to preventing XSS attacks, so we should probably make a dedicated spec.